cmd/rootMain.go

```diff
@@ -118,17 +119,20 @@ func rootMain(cmd *cobra.Command, args []string) exitCode {
 		return exitCode{1}
 	}
 	srv := server.New(mountpath, socketpath, plugins, serverOpts)
-	if err := srv.Start(); err != nil {
+	successfullyLoadedPlugins, err := srv.Start()
+	if err != nil {
 		cmdutil.ErrPrintf("Unable to start server: %v\n", err)
 		return exitCode{1}
 	}
 	defer srv.Stop()
 
-	if plugin.IsInteractive() {
-		cmdutil.Println(`Welcome to Wash!
-  Wash includes several built-in commands: ls, wexec, find, meta, tail.
-  See commands run with wash via 'whistory', and logs with 'whistory <id>'.
-Try 'help'`)
+	if plugin.IsInteractive() && successfullyLoadedPlugins {
+		var randomPlugin string
+		for plugin := range plugins {
+			randomPlugin = plugin
+			break
+		}
+		cmdutil.Println(fmt.Sprintf("Welcome to Wash! Try 'ls' then 'docs <child>' (e.g. 'docs %v')", randomPlugin))
```

Comment:
Reason for this instead of `docs .` is so they get used to doing `ls` => `docs <child>` when navigating through Wash (and being comfortable w/ `docs` as the main source of truth for entries). It is also a useful way to introduce `stree <plugin_root>`. If this seems OK, I can update the registry's description.

Reply:
Here's an example first-time UX:
```
Eniss-MacBook-Pro:wash enisinan$ ./wash
Do you use aws (y/n)? y
docker? y
gcp? y
kubernetes? y
The aws, docker, gcp, kubernetes core plugins have been enabled.
You can disable them by modifying the 'plugins' key in your config
file (~/.puppetlabs/wash/wash.yaml), and then restarting the shell

Welcome to Wash! Try 'ls' then 'docs <child>' (e.g. 'docs aws')
wash . ❯ ls
aws/
docker/
gcp/
kubernetes/
wash . ❯ docs aws
This is the AWS plugin root. The AWS plugin reads the AWS_SHARED_CREDENTIALS_FILE
environment variable or $HOME/.aws/credentials and AWS_CONFIG_FILE environment
variable or $HOME/.aws/config to find profiles and configure the SDK. See
https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/guide_credentials_profiles.html
for more details on how to setup AWS profiles.

You can limit the loaded profiles by adding

aws:
  profiles: [profile_1, profile_2]

to Wash’s config file.

The AWS plugin currently supports EC2 and S3. IAM roles are supported when configured
as described here. Note that currently region will also need to be specified with the
profile.

If using MFA, Wash will prompt for it on standard input. Credentials are valid for 1 hour.
They are cached under wash/aws-credentials in your user cache directory so they can be
re-used across server restarts. Wash may have to re-prompt for a new MFA token in response
to navigating the Wash environment to authorize a new session.

SUPPORTED ACTIONS
* list
    - ls aws
        Type 'docs <child>' to view an ls'ed child's documentation
    - cd aws
        The 'W' environment variable contains the Wash root. Use 'cd $W' to
        return to it.
    - stree aws
        Gives you a high-level overview of the kinds of things that you will
        encounter when you 'cd' and 'ls' through this entry.
    - find aws <expression>
        Prints all children that satisfy <expression>. Type 'find --help' to
        see all the things that you can filter on.
    - (anything else that works with directories [e.g. 'tree'])

Type 'help' to see a list of all the built-in Wash commands.
```

Reply:
Here's an example of an entry that includes the `SUPPORTED ATTRIBUTES` section:
```
wash . ❯ docs aws/wash/resources/ec2/instances/i-04621c13583930e6c/
This is an EC2 instance. Its Exec action uses SSH. It will look up port, user,
and other configuration by exact hostname match from default SSH config files.
If present, a local SSH agent will be used for authentication. Lots of SSH
configuration is currently omitted, such as global known hosts files, finding
known hosts from the config, identity file from config… pretty much everything
but port and user from config as enumerated in
https://github.com/kevinburke/ssh_config/blob/0.5/validators.go. The known hosts
file will be ignored if StrictHostKeyChecking=no, such as in

Host *.compute.amazonaws.com
  StrictHostKeyChecking no

SUPPORTED ATTRIBUTES
* mtime (last modified time) -- 2020-01-01 22:15:53 +0000 UTC
* crtime (creation time) -- 2020-01-01 22:15:52 +0000 UTC

The attributes are a subset of the entry's metadata, which contains everything
you ever need to know about the entry. You can use
    meta aws/wash/resources/ec2/instances/i-04621c13583930e6c/
to view the metadata and
    meta --attribute aws/wash/resources/ec2/instances/i-04621c13583930e6c/
to view the (partial) metadata.

SUPPORTED ACTIONS
* delete
    - delete aws/wash/resources/ec2/instances/i-04621c13583930e6c/
* exec
    - wexec aws/wash/resources/ec2/instances/i-04621c13583930e6c/ <command> <args...>
        e.g. wexec aws/wash/resources/ec2/instances/i-04621c13583930e6c/ uname
* list
    - ls aws/wash/resources/ec2/instances/i-04621c13583930e6c/
        Type 'docs <child>' to view an ls'ed child's documentation
    - cd aws/wash/resources/ec2/instances/i-04621c13583930e6c/
        The 'W' environment variable contains the Wash root. Use 'cd $W' to
        return to it.
    - stree aws/wash/resources/ec2/instances/i-04621c13583930e6c/
        Gives you a high-level overview of the kinds of things that you will
        encounter when you 'cd' and 'ls' through this entry.
    - find aws/wash/resources/ec2/instances/i-04621c13583930e6c/ <expression>
        Prints all children that satisfy <expression>. Type 'find --help' to
        see all the things that you can filter on.
    - (anything else that works with directories [e.g. 'tree'])
* signal
    - signal <signal> aws/wash/resources/ec2/instances/i-04621c13583930e6c/
        e.g. signal start aws/wash/resources/ec2/instances/i-04621c13583930e6c/

Type 'help' to see a list of all the built-in Wash commands.

SUPPORTED SIGNALS
* start
    Starts the EC2 instance
* stop
    Stops the EC2 instance
* hibernate
    Hibernates the EC2 instance
* restart
    Reboots the EC2 instance
* terminate
    Terminates the EC2 instance
```

Reply:
Hmm, I guess one advantage of just saying `Try 'docs .'` is that people could see all the `list` examples, so they'd be exposed to `stree` a bit earlier (without having to scrounge through detailed plugin root description). But I'm not sure how likely a long description would deter people from reading the `list` stuff -- my guess "not very" since I think it is expected that a `docs` command's output could be verbose.

Reply:
I'd like to include `help` still too. It's not obvious that that should work.

Reply:
I think I'm going to go back to "Try 'docs .'" --  biggest reason is b/c `find`'s being moved over to the Attributes/Metadata section so a first-time user won't see it being mentioned for a while. I'll mention `help` as part of the registry's description (which I'll also reword per your suggestion). Does that work? I like that better b/c `docs` is still the entry-point, but `help` is also mentioned early-on.

Reply:
Ok, updated. You'll find `help` at the end of `docs <registry>`'s output